docs/zh-CN/01.云沙箱/04.功能说明/04.文件系统/03.上传和下载文件.md

```
@@ -2,6 +2,8 @@
 
 上传和下载地址适合浏览器直传、把结果文件交给未持有 SDK 鉴权信息的环境，或把文件交给后续系统处理。小文件、二进制内容、流式内容和 Agent 生成代码优先使用 `sandbox.files.write()` 和 `sandbox.files.read()`。
 
+> **大文件说明**：单个 HTTP 请求正文最大为 32 MiB。上传接近或超过该大小的文件时，请使用 OSS 中转或分块直传；完整方案和示例参见[FC 云沙箱大文件上传最佳实践：OSS 中转与分块直传](../../05.最佳实践/13.大文件上传最佳实践.md)。
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)